apps/Info.html

```diff
@@ -60,9 +60,12 @@
 				<li class="nav-item link">
 					<a class="nav-link" href="landing/landing.html"> <i class="fas fa-home"></i> Home</a>
 				</li>
-				<li class="nav-item link active">
+				<li class="nav-item link">
 					<a class="nav-link" href="table.html"> <i class="fas fa-list-ul"></i> Slides</a>
 				</li>		
+				<li class="nav-item active link">
+				<a class="nav-link" href="info.html"> <i class="fas fa-info-circle"></i> Info</a>
```

Comment:
I'm getting a 404 on all links to info.html; it appears to be case sensitive. we can rename this file to info.html or change the links back to Info.html

Reply:
```suggestion
				<a class="nav-link" href="Info.html"> <i class="fas fa-info-circle"></i> Info</a>
```